README.md

```diff
@@ -4,6 +4,61 @@ QueueNado
 `QueueNado` is a collection of C++ implemntations of ZMQ fast queue patterns (Push/Pull, request/reply, pub/sub, dealer/router) for C++
 
 
+## Requirements
+1. [g3log](https://github.com/KjellKod/g3log)
+2. [g3sinks](https://github.com/KjellKod/g3sinks)
+3. [FileIO](https://github.com/LogRhythm/FileIO)
+4. [ZMQ v3.2.5](http://zeromq.org/)
+5. [ZLIB](http://www.zlib.net)
+6. [ZLIB](http://www.zlib.net)
+Most Linux distributions carry zlib. 
+Example: In Ubuntu it can be installed with 
+```sudo apt-get install zlib1g-dev``` 
+7. [StopWatch](https://github.com/LogRhythm/StopWatch)
+8. [DeathKnell](https://github.com/LogRhythm/DeathKnell)
+9. [tcmalloc](https://github.com/gperftools/gperftools)
+The creators of tcmalloc recommends installing ` libunwind`   to avoid encountering a  deadlock bug during stack unwinding. 
+10. [boost](http://www.boost.org/doc/libs/1_63_0/index.html)
+For installing boost please follow their [instructions](http://www.boost.org/doc/libs/1_63_0/more/getting_started/unix-variants.html). 
+
+
+## Build and Install
+Example of installation
+```
+cd Queuenado
+cd 3rdparty
+unzip gtest-1.7.0.zip
+cd ..
+mkdir build
+cd build
+cmake ..
+make -j
+```
+
+### Executing the unit tests
+```
+./UnitTestRunneer
```

Comment:
This will need to be run as sudo. UnitTestRunner should be linked to use the new library built in the same directory. One way to do that would be to include an rpath to the local build directory.